LocalMultiplayerAgent/MultiplayerSettings.json

```diff
@@ -1,51 +1,43 @@
 {
-    "RunContainer": false,
```

Comment:
changes to this file should be reverted

Reply:
reverted. I think I messed up from my previous revert.